lib/features/voucher/data/datasources/voucher_remote_data_source.dart

```diff
@@ -3,23 +3,23 @@ import 'package:coffeecard/core/extensions/either_extensions.dart';
 import 'package:coffeecard/core/network/network_request_executor.dart';
 import 'package:coffeecard/features/voucher/data/models/redeemed_voucher_model.dart';
 import 'package:coffeecard/features/voucher/domain/entities/redeemed_voucher.dart';
-import 'package:coffeecard/generated/api/coffeecard_api.swagger.dart';
+import 'package:coffeecard/generated/api/coffeecard_api_v2.swagger.dart';
 import 'package:fpdart/fpdart.dart';
 
 class VoucherRemoteDataSource {
-  final CoffeecardApi apiV1;
+  final CoffeecardApiV2 apiV2;
```

Comment:
just call the field name "api". we use the distinctions in the name when we use both versions in the same class